pages/parent/signup.tsx

```diff
@@ -421,55 +401,17 @@ export default function ParticipantInfo({
             body: JSON.stringify(userData),
         };
         const response = await fetch("api/user", request);
+        console.log(response);
         const updatedUserData = await response.json();
         return updatedUserData;
     }
 
-    const clearLocalStorage = () => {
-        setParticipantFirstName("");
-        setParticipantLastName("");
-        setDateOfBirth("");
-        setAddress1("");
-        setAddress2("");
-        setCity("");
-        setParticipantProvince(DEFAULT_PROVINCE);
-        setPostalCode("");
-        setSchool("");
-        setGrade("");
-        setHasLearningDifficulties(false);
-        setHasPhysicalDifficulties(false);
-        setHasSensoryDifficulties(false);
-        setHasOtherDifficulties(false);
-        setOtherDifficulties("");
-        setParticipantDifficulties([]);
-        setSpecialEd(false);
-        setPhysiotherapy(false);
-        setSpeechTherapy(false);
-        setOccupationalTherapy(false);
-        setCounseling(false);
-        setArtTherapy(false);
-        setHasOtherTherapy(false);
-        setOtherTherapy("");
-        setParticipantTherapy([]);
-        setGuardianExpectations("");
-        setMedication("");
-        setAllergies("");
-        setEmergFirstName("");
-        setEmergLastName("");
-        setEmergNumber("");
-        setEmergRelationship("");
-        setHeardFromFriendsAndFam(false);
-        setHeardFromFlyers(false);
-        setHeardFromEmail(false);
-        setHeardFromSocialMedia(false);
-        setHeardFromOther(false);
-        setHeardFromOptions([]);
-    };
-
+    //Form is finished
     async function updateUserAndClearForm() {
+        //Save the user
         const updatedUser = await updateUser();
         if (updatedUser) {
-            clearLocalStorage();
+            localStorage.clear();
```

Comment:
As mentioned in the discord, this may cause issues with the session auth. 

However, there must be a better way to do this than what we currently have (which actually misses some cases)

Reply:
I guess this is fine, but we would want to keep a note in case this cases a regression in the future.